openaz-xacml/src/main/java/org/apache/openaz/xacml/std/jaxp/JaxpResponse.java

```diff
@@ -102,18 +103,16 @@ public static JaxpResponse load(File fileXmlResponse) throws ParserConfiguration
             return null;
         }
 
-        NodeList nodeListRoot = document.getChildNodes();
-        if (nodeListRoot == null || nodeListRoot.getLength() == 0) {
-            logger.warn("No child elements of the XML document");
-            return null;
-        }
-        Node nodeRoot = nodeListRoot.item(0);
-        if (nodeRoot == null || nodeRoot.getNodeType() != Node.ELEMENT_NODE) {
-            logger.warn("Root of the document is not an ELEMENT");
-            return null;
-        }
+        Node nodeRoot = document.getDocumentElement();
 
-        JAXBContext context = JAXBContext.newInstance(ResponseType.class);
+        if(context == null) {
```

Comment:
I don't feel it's good, we use the JaxpResponse to load XACML response string to an Response object and we construct the ResponseType object by our own unmarshal code. So it make sense to use the class without using it's own load method.